stores/sql/main.go

```diff
@@ -1674,10 +1666,39 @@ func RecordHostScans(ctx context.Context, tx sql.Tx, scans []api.HostScan) error
 	return nil
 }
 
-func RemoveContractSet(ctx context.Context, tx sql.Tx, contractSet string) error {
-	_, err := tx.Exec(ctx, "DELETE FROM contract_sets WHERE name = ?", contractSet)
+func RecordPriceTables(ctx context.Context, tx sql.Tx, priceTableUpdates []api.HostPriceTableUpdate) error {
```

Comment:
I don't see RecordPriceTables in my version of its-happening. Did that sneak back in?